AElf.OS/Network/INetworkService.cs

```diff
@@ -0,0 +1,19 @@
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using AElf.Common;
+using AElf.Kernel;
+
+namespace AElf.OS.Network
+{
+    public interface INetworkService
+    {
+        void AddPeer(string address);
```

Comment:
if network service has the same method with network manager, we do not need it.

just check the different between manager and service

Reply:
In this case, as said in other comments, we don't really need anything that's a "Manager". In my opinion we can use the "NetworkService" (used internally by our RPC and handlers in our system) and "ServerSingleton" (maybe not final name), these two are enough.